trio/_subprocess.py

```diff
@@ -815,6 +815,56 @@ async def open_process(
+
+            After construction, you can interact with the child process by writing data to its
+            `~trio.Process.stdin` stream (a `~trio.abc.SendStream`), reading data from its
+            `~trio.Process.stdout` and/or `~trio.Process.stderr` streams (both
+            `~trio.abc.ReceiveStream`\s), sending it signals using `~trio.Process.terminate`,
+            `~trio.Process.kill`, or `~trio.Process.send_signal`, and waiting for it to exit
+            using `~trio.Process.wait`. See `trio.Process` for details.
+
+            Each standard stream is only available if you specify that a pipe should be created
+            for it. For example, if you pass ``stdin=subprocess.PIPE``, you can write to the
+            `~trio.Process.stdin` stream, else `~trio.Process.stdin` will be ``None``.
+
+            Unlike `trio.run_process`, this function doesn't do any kind of automatic
+            management of the child process. It's up to you to implement whatever semantics you
+            want.
+
+            Args:
+              command (list or str): The command to run. Typically this is a
+                  sequence of strings such as ``['ls', '-l', 'directory with spaces']``,
+                  where the first element names the executable to invoke and the other
+                  elements specify its arguments. With ``shell=True`` in the
+                  ``**options``, or on Windows, ``command`` may alternatively
+                  be a string, which will be parsed following platform-dependent
+                  :ref:`quoting rules <subprocess-quoting>`.
+              stdin: Specifies what the child process's standard input
+                  stream should connect to: output written by the parent
+                  (``subprocess.PIPE``), nothing (``subprocess.DEVNULL``),
+                  or an open file (pass a file descriptor or something whose
+                  ``fileno`` method returns one). If ``stdin`` is unspecified,
+                  the child process will have the same standard input stream
+                  as its parent.
+              stdout: Like ``stdin``, but for the child process's standard output
+                  stream.
+              stderr: Like ``stdin``, but for the child process's standard error
+                  stream. An additional value ``subprocess.STDOUT`` is supported,
+                  which causes the child's standard output and standard error
+                  messages to be intermixed on a single standard output stream,
+                  attached to whatever the ``stdout`` option says to attach it to.
+              **options: Other :ref:`general subprocess options <subprocess-options>`
+                  are also accepted.
+
+            Returns:
+              A new `trio.Process` object.
+
+            Raises:
+              OSError: if the process spawning fails, for example because the
+                 specified command could not be found.
```

Comment:
Is this docstring format correct? You use the plain sphinx format elsewhere in this PR.

Reply:
This is a copy-paste of the docstring at line 346 for `_open_process` - but maybe that one is incorrectly formatted? Or all the other ones I copy-pasted from the docs are?

Reply:
TBH as long as sphinx renders this fine then whatever. And it does seem to.